src/language-js/parser-babylon.js

```diff
@@ -58,17 +58,21 @@ function createParse(parseMethod, extraPlugins) {
     try {
       ast = babel[parseMethod](text, babelOptions({}, extraPlugins));
     } catch (error) {
-      throw createError(
-        // babel error prints (l:c) with cols that are zero indexed
-        // so we need our custom error
-        error.message.replace(/ \(.*\)/, ""),
-        {
-          start: {
-            line: error.loc.line,
-            column: error.loc.column + 1
+      if (error.loc) {
```

Comment:
Can Babel throw an error without `loc`?

Reply:
It didn't, babel itself targets newer node version than prettier, it thrown `includes is not a function` error, you can check the discussion flow at two months ago when I tried to included `js_expression` parser.

Reply:
Of course, I'm fine to revert it if you want.

Reply:
Yes, let's try reverting it. It doesn't seem to belong to this PR.

Reply:
@thorn0 Done, shall we merge this PR then?